hw_1/task_1.py

```diff
@@ -0,0 +1,13 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) 
+
+print('введите первое число: ')
+a = int(input())
```

Comment:
Необязательно использовать функцию `print` для вывода сообщения, приглашения к вводу. Это сообщение можно поместить внутрь круглых скобок функции `input`.
```suggestion
a = int(input('введите первое число: '))
```